4.sc_Morphology_Neural_Network_MLP_Model/1.model_training/scripts/Multi_Classifier_Call.sh

```diff
@@ -6,7 +6,7 @@
 #SBATCH --partition=amem
 #SBATCH --qos=mem
 #SBATCH --account=amc-general
-#SBATCH --time=72:00:00
+#SBATCH --time=36:00:00
```

Comment:
Didn't know slurm syntax was specified like a bash comment, not sure if I like that

Reply:
Haha! Yes! I love this comment @MattsonCam . It feels...off. But nerveless, this is how it is done.